### ISSUE TITLE
Static/fuzz seen when using remote debugging

### ISSUE THREAD

**Issue by patricknelson** [googlevr/chromium-webar#20]
Hi, I'm not sure if this is an issue specifically with this particular build of Chromium, but I wanted to get this out there in case it is and anyone else is experiencing the bug. Basically, when you connect the phone (in my case, ASUS ZenFone AR) and start a remote debugging session in desktop version of Chrome, this is what you see (below). Note that the same static does not occur when using [Vysor](https://chrome.google.com/webstore/detail/vysor/gidgenkbbabolejbgbpnhbimgjbffefm?hl=en-US) (which displays/controls the whole phone, not just the contents of Chrome/Chromium itself).

Actual screenshot during static/fuzz in session:

![actual screenshot](https://user-images.githubusercontent.com/4269377/30339411-c844effc-97a3-11e7-876a-f6d030d01398.jpg)

What is displayed in DevTools:

![devtools static](https://user-images.githubusercontent.com/4269377/30339401-c421fa28-97a3-11e7-9c9a-815c6ebb8855.gif)

Does anyone else experience this?

**Comment by jsantell** [comment]
I've gotten that as well, I think due to the underlying changes and hacks in Chromium/WebGL/video rendering

**Comment by patricknelson** [comment]
Yeah. If it helps, I've also noticed this on another app that utilizes AR.js to stream video to a `<video>` element, whereas this renders it to `<canvas>` via `THREE.VideoTexture`.
